packages/gestalt/src/Borders.css

```diff
@@ -68,43 +68,43 @@ html[dir="rtl"] .borderLeft {
 }
 
 .pill {
-  border-radius: 999px;
+  border-radius: var(--rounding-pill);
```

Comment:
we should also tokenize border-radius: 50%, no?

Reply:
Yeah I wasn't sure if percents are okay, but seems fine